doc/changes/latest.rst

```diff
@@ -110,3 +110,5 @@ Changes
 - :bdg-dark:`Code` Reorder condition in internal call of :func:`nilearn.image.resample_img` to skip checking of array values if interpolation is ``nearest`` (:gh:`4571` by `Jason Kai`_).
 
 - :bdg-dark:`Code` Remove redundant sorting of ``np.unique(data)`` in internal call of :func:`nilearn.image.resample_img` when checking array values (:gh:`4571` by `Jason Kai`_).
+
+- :bdg-secondary:`Maint` Refactored :func:`nilearn.plotting.displays._projectors.add_graph` to reduce its maxixmum length (:gh:`4635` by `Anupriya Kumari`_).
```

Comment:
```suggestion
- :bdg-secondary:`Maint` Refactor `nilearn.plotting.displays._projectors.add_graph` to reduce its maxixmum length (:gh:`4635` by `Anupriya Kumari`_).
```

Reply:
removing the `:func:` that causes the doc build to crash (this function is not part of the public API so there is no official reference to it)